modules/nw-enabling-a-provisioning-network-after-installation.adoc

```diff
@@ -12,16 +12,16 @@ In {product-title} 4.8 and later, you can enable a `provisioning` network after
 
 .Prerequisites
 
-. The `provisioning` network must exist.
-. The `provisioning` network must be enabled.
-. The cluster nodes must be connected to the `provisioning` network using the same network interface on both worker nodes and control plane nodes.
-. The cluster nodes must be homogeneous. If the cluster nodes use different network interface names for the same network interface order, such as `eth0` and `eno1` for the first network interface, the procedure fails.
+. A dedicated physical network must exist, connected to all worker and control plane nodes.
```

Comment:
The prerequisites should be an unnumbered bullet list. Do you mind updating while you're in here?